.github/workflows/build_cmake.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     - name: Build
       working-directory: ${{ github.workspace }}/dlib/test
-      run: cmake --build ${{ env.build_dir }} --config ${{ env.config }} --target dtest
+      run: cmake --build ${{ env.build_dir }} --config ${{ env.config }} --target dtest --parallel 8
```

Comment:
This appears to decrease the Ubuntu build time from ~23 minutes to ~15 minutes.

Didn't test any other values than 8, at least just yet.

Reply:
There's a lot of variance, though: the next run with `--parallel 8` was less than 12 minutes.

Reply:
(I mean just the build time.)

Reply:
Sweet.  If github let's us do it then we should :D